ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/browser/DisposableJsFunction.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Project: RCS - Rail Control System
+ *
+ * © Copyright by SBB AG, Alle Rechte vorbehalten
+ */
+package ch.sbb.scion.rcp.microfrontend.browser;
+
+/**
+ * A JavaScript function which should be disposed of to free up resources when no longer needed.
+ */
+public interface DisposableJsFunction {
```

Comment:
BTW, why it isn't just `Disposable`?  With its API, I don't see any benefit of restricting the implementations by calling this parent interface a "JS function".

If we need an interface for JS functions, let's create it separately (extending this one). Right now it is a mix of very different things (like in the example below):

```java
public interface SeniorDeveloperCat {

    void meow();
}